src/ShallowTraversal.js

```diff
@@ -18,8 +21,17 @@ import {
 export function childrenOfNode(node) {
   if (!node) return [];
   const maybeArray = propsOfNode(node).children;
+  const recursiveWithout = array =>
+    map(without(array, false, undefined, null), value => {
+      if (isArray(value)) {
+        return recursiveWithout(value);
+      }
+      return value;
+    });
+
+  const filtered = isArray(maybeArray) ? recursiveWithout(maybeArray) : maybeArray;
   const result = [];
-  React.Children.forEach(maybeArray, child => result.push(child));
+  React.Children.forEach(filtered, child => result.push(child));
```

Comment:
@nishitaniyuki yep! Update your code and then push the changes to github. this PR should update automatically.

When you update your code, it's best for a change like this if you can make all of your changes into a single atomic commit. Let me know if you need any extra help with this.